lib/spack/llnl/util/filesystem.py

```diff
@@ -585,24 +585,23 @@ def __str__(self):
 
 
 def find_system_libraries(args, shared=True):
-    """Searches the usual system library locations for the libraries
-    specified in args.
+    """Searches the system library locations libraries.
```

Comment:
> First line (summary) in a docstring is supposed to be one line

Is this a rule of thumb or a hard requirement? Guess I've never heard it before. Makes sense though.